### ISSUE TITLE
[OUDS] Tag icon resising error in

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/Orange-Boosted-Bootstrap#3295]
### Prerequisites

- [x] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [x] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

I got an issue with the font size change on the tag non functional icon with the ones showing the settings:
https://web.unified-design-system.orange.com/orange/docs/components/tags/#non-functional-1

<img width="651" height="397" alt="Image" src="https://github.com/user-attachments/assets/01a02f92-dc80-4fdf-9df3-64a69513d544" />

### Reduced test cases

the icon should be fully visible

### What operating system(s) are you seeing the problem on?

macOS

### What version of Boosted are you using?

0.6

**Comment by B3nz01d** [comment]
Finalized during sprint # 29
